gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/converter/UserConverter.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.rest.api.service.converter;
+
+import io.gravitee.common.util.Maps;
+import io.gravitee.repository.management.model.User;
+import io.gravitee.repository.management.model.UserStatus;
+import io.gravitee.rest.api.model.NewExternalUserEntity;
+import io.gravitee.rest.api.model.UserEntity;
+import io.gravitee.rest.api.model.UserMetadataEntity;
+import io.gravitee.rest.api.service.common.GraviteeContext;
+import java.util.List;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author GraviteeSource Team
+ */
+@Component
```

Comment:
Is there any benefit in using Spring tooling with `@Component` instead of an `abstract class` containing only `static` methods? It would be easier to use as it would require no `@autowired` nor dedicated mock in the unit tests

Reply:
I followed the pattern of existing converters.
The fact you dont require a mock can be an advantage, or be a disadvantage to not be able to mock it.
I think it's better to keep the same pattern for all converters, and then, we may refactor *all* converters, if relevant.
WDYT @gaetanmaisse ?

Reply:
>I think it's better to keep the same pattern for all converters, and then, we may refactor all converters, if relevant.

Agreed on that one so go for it 🚀 

>The fact you don't require a mock can be an advantage, or be a disadvantage to not be able to mock it.

Really not convinced by this one, especially for converters, as for me you should always call the real method and not tweak it and end up with a Frankenstein object 🙈